app/bundles/ReportBundle/Resources/views/Graph/Table.html.twig

```diff
@@ -21,25 +21,27 @@
                     </thead>
                     <tbody>
                         {% for rowKey, row in graph.data %}
-                        <tr>
-                            {% for cellName, cell in row %}
-                                {% if graph.data[0].id is defined and 'title' == cellName and graph.link is defined %}
-                                    <td>
-                                        <a href="{{ path(graph.link, {'objectAction': 'view', 'objectId': row.id}) }}" data-toggle="ajax">
-                                            {{ cell }}
-                                        </a>
-                                    </td>
-                                {% elseif 'id' != cellName %}
-                                    <td>
-                                        {% if cell is numeric %}
-                                            {{ cell }}
-                                        {% else %}
-                                            {{ assetMakeLinks(cell) }}
+                            {% if row['id'] is not null %}
+                                <tr>
+                                    {% for cellName, cell in row %}
+                                        {% if graph.data[0].id is defined and 'title' == cellName and graph.link is defined %}
+                                            <td>
+                                                <a href="{{ path(graph.link, {'objectAction': 'view', 'objectId': row.id}) }}" data-toggle="ajax">
+                                                    {{ cell }}
+                                                </a>
+                                            </td>
+                                        {% elseif 'id' != cellName %}
+                                            <td>
+                                                {% if cell is numeric %}
+                                                    {{ cell }}
+                                                {% else %}
+                                                    {{ assetMakeLinks(cell) }}
+                                                {% endif %}
+                                            </td>
                                         {% endif %}
```

Comment:
I don't quite understand this change. The `assetMakeLinks` function is called only if the `cellName` is NOT equal to `id`. So it cannot be the `id` part that is `null`, right? Can you provide a functional test that would show the data that are causing the error in the linked issue?

Reply:
Sometimes there is a pagehits record with pageId equal to null. This produces null results and a 500 error.

Reply:
@escopecz, I added functional tests.